openslides/agenda/static/js/agenda/agenda.js

```diff
@@ -115,9 +115,32 @@ angular.module('OpenSlidesApp.agenda.site', ['OpenSlidesApp.agenda'])
     };
 })
 
-.controller('ItemDetailCtrl', function($scope, Agenda, User, item) {
+.controller('ItemDetailCtrl', function($scope, $http, Agenda, User, item) {
     Agenda.bindOne(item.id, $scope, 'item');
     User.bindAll({}, $scope, 'users');
+    $scope.speaker = {};
+
+    // close/open list of speakers
+    $scope.closeList = function (item_id, bool) {
```

Comment:
you do not use 'item_id'.

Could you rename 'bool' to a name that says what the value could be, and not which type it is?